client/components/public/Gear.jsx

```diff
@@ -82,152 +68,136 @@ const prices = {
-                  <center>
-                  { buyButton }
-                </center>
-                </Grid.Column>
+              <Grid stackable columns={2} style={{flexWrap: "wrap-reverse"}}>
                 <Grid.Column width={10}>
-                  <Header id="title" as="h2">{this.state.title}</Header>
-                  <p> {this.state.price}</p>
-                  <p>Size range: {this.state.size}</p>
-                  <p>Material: {this.state.materials}</p>
-                  <Header as="h2">Colors:</Header>
-                  {
-                    Object.keys(this.state.colors).map((idx) => {
+                    {Object.keys(this.state.colors).map((idx) => {
                       let color_name = this.state.colors[idx];
                       return (
-                        <div className="gearitem ui card" key={idx} style={{verticalAlign: "top", display: "inline-block", marginLeft: "1em"}}>
-                          <img style={{maxHeight: "100%", maxWidth: "100%", verticalAlign: "bottom"}} src={`${s3_prefix}${this.state.code}_${color_name.toLowerCase().replaceAll(" ", "")}.jpg`}/>
-                          <Card.Content>
-                            <Card.Description>
+                        <div className="gearitem ui card" key={idx} style={{verticalAlign: "top", display: "inline-block", marginRight: "10px", marginTop: "0"}}>
+                          <img style={{maxHeight: "100%", maxWidth: "100%", verticalAlign: "bottom"}} src={this.getImageURL(this.state.code, color_name)}/>
+                          <Card.Content style={{padding: "10px"}}>
                               { color_name }
-                            </Card.Description>
-
                           </Card.Content>
-
                         </div>
                       );
                     }) 
                   }
                   <img width="100%" src={this.state.swatches}></img>
                 </Grid.Column>
-                
+                <Grid.Column width={6}>
+		              <Header id="title" as="h2">{this.state.title}</Header>
+                  <p> {this.state.price}</p>
+                  <p>Size range: {this.state.size}</p>
+                  <p>Material: {this.state.materials}</p>
+		              { buyButton }
+                </Grid.Column>
             </Grid>
             </Modal.Content>
         </Modal>
 
         <Segment basic>
           <PuzzlePageTitle title="Gear" />
+          <Header as="h2">PLEASE NOTE:</Header>
+          SHIRTS ARRIVE <b>AFTER</b> EVENT and are <b>NOT</b> AVAILABLE FOR PICKUP AT EVENT.
+          <br /><br />
           Gear store closes { gearSaleEnd }.
           <br />
-          Shirts will be ordered on Monday, April 17 and assuming no supply chain delays, should be shipped out or ready for pick-up the week of April 24.
+          Shirts will be ordered on Monday, April 22, 2024, and assuming no supply chain delays, should be shipped out or ready for pick-up the week of May 6, 2024.
```

Comment:
I'm still surprised to see this diff since I thought I changed this already. May be worth trying to rebase but since the changes are identical maybe it'll merge cleanly.